src/ONE.Models.CSharp/Common/Configuration/ConfigurationTypeConstants.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ONE.Models.CSharp.Common.Configuration
```

Comment:
Having Common.Configuration in the namespace indicates that these are models used by the Common.Configuration MSApp which is not the case.  I might suggest a One.Models.CSharp.Constants namespace and then it can include classes like ConfigurationTypes, ActivityTypes, Roles, etc.

Reply:
I think we just need to agree on a standard.  I just refactored it from what you are suggesting to how it is currently implemented through a separate conversation.

Reply:
Perhaps an alignment meeting is in order.

Reply:
Moved to Constants